scalameter-core/src/main/scala/org/scalameter/PicklerBasedKey.scala

```diff
@@ -5,20 +5,23 @@ import org.scalameter.picklers.Pickler
 
 /* A mixin for keys whose values require pickler based serialization. */
 trait PicklerBasedKey[T] extends Serializable {
-  /* Name of the key that will be serialized. */
+  /** Name of the key that will be serialized. */
   def fullName: String
 
-  /* Pickler used to deserialize value to which the key refers. */
+  /** Pickler used to deserialize value to which the key refers. */
   def pickler: Pickler[T]
 
+  /** Indicates if a key should be serialized. */
```

Comment:
Indicates if a key should be skipped during serialization - if `true`,
    the key will not be serialized.